src/quantum/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '0.11.2006.4'
+VERSION = '0.12.2010.1901'
```

Comment:
There is no intent to publish this version. Once the 'create' command is added too, an entry will be added to HISTORY.rst, and this version might be updated again.